pkg/types/alibabacloud/metadata.go

```diff
@@ -2,7 +2,10 @@ package alibabacloud
 
 // Metadata contains Alibaba Cloud metadata (e.g. for uninstalling the cluster).
 type Metadata struct {
-	Region          string `json:"region"`
-	ResourceGroupID string `json:"resourceGroupID"`
-	ClusterDomain   string `json:"clusterDomain"`
+	Region          string   `json:"region"`
+	ResourceGroupID string   `json:"resourceGroupID"`
+	VpcID           string   `json:"vpcID"`
+	VSwitchIDs      []string `json:"vswitchIDs"`
+	PrivateZoneID   string   `json:"privateZoneID"`
```

Comment:
In what way are these values going to be used while destroying the cluster?

Reply:
I have to admit that these are not used, I will  delete it.